tutorials/docs/workflow.md

```diff
@@ -0,0 +1,96 @@
+# WhizzML: Training and Predicting Workflows
+
+## The detailed Training Workflow
+
+The basic process to create a model in BigML goes through
+several steps. In each step, a new resource is generated.
+You can see a sketch of the process in the following image.
+
+![Training steps](images/training_steps.png)
+
+
+### Step 0: uploading your data to create a **Source**
+
+The first step is creating a **Source** by uploading your data.
+
+This means that you'll need to use the **create-source** call
+to create a **Source** object. The only mandatory argument
+is the path to your data, and the rest of arguments that can be found in the
+[API Documentation](https://bigml.com/api/sources) are
+optional.
```

Comment:
this should probably mention that "the path to your data" is in fact a remote URL and the value of the "remote" key...